src/VirtoCommerce.DemoSolutionFeaturesModule.Core/Models/Order/DemoCustomerOrder.cs

```diff
@@ -16,5 +19,10 @@ public override object Clone()
 
             return result;
         }
+
+        private LineItem[] GetUsualItems()
```

Comment:
What is it?

Reply:
https://rules.sonarsource.com/csharp/RSPEC-2365

Reply:
Changes will be reverted